packages/workflow/src/workflow.ts

```diff
@@ -1322,7 +1334,9 @@ export function setDefaultSignalHandler(handler: DefaultSignalHandler | undefine
  * Updates this Workflow's Search Attributes by merging the provided `searchAttributes` with the existing Search
  * Attributes, `workflowInfo().searchAttributes`.
  *
- * For example, this Workflow code:
+ * Search attributes can be upserted using either SearchAttributes (deprecated) or SearchAttributeUpdatePair[]
```

Comment:
I'd only mention the new approach. Or at very least, mention it first.

Reply:
Fair, removed the old approach from the doc.